synapse/handlers/room_member.py

```diff
@@ -871,11 +842,39 @@ def _ask_id_server_for_third_party_invite(
             "sender_avatar_url": inviter_avatar_url,
         }
 
+        # Add the identity service access token to the JSON body and use the v2
```

Comment:
I feel like this bit (the switch to /v2/ for these APIs) should probably be a separate PR? It's not mentioned in the newsfile, and it seems like a good candidate to stand alone (I don't think it is required before we land the other half?)

Reply:
v2 lookup won't work without `id_access_token` support, but could merge it even though it would fail 100% of the time?

Reply:
Ah I see, you meant the 3PID-invite related things. Yes, that can be moved out.

Moved to https://github.com/matrix-org/synapse/pull/5979